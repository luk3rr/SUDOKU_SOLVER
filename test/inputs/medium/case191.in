006001002 230060000 000000050 008300010 000800036 060400905 000004003 020007069 047009800 
