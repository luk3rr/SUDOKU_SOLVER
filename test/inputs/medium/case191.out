Solving the following grid:
0 0 6 | 0 0 1 | 0 0 2 
2 3 0 | 0 6 0 | 0 0 0 
0 0 0 | 0 0 0 | 0 5 0 
------+-------+------
0 0 8 | 3 0 0 | 0 1 0 
0 0 0 | 8 0 0 | 0 3 6 
0 6 0 | 4 0 0 | 9 0 5 
------+-------+------
0 0 0 | 0 0 4 | 0 0 3 
0 2 0 | 0 0 7 | 0 6 9 
0 4 7 | 0 0 9 | 8 0 0 


Solution found :')

5 8 6 | 7 4 1 | 3 9 2 
2 3 9 | 5 6 8 | 1 4 7 
1 7 4 | 9 2 3 | 6 5 8 
------+-------+------
9 5 8 | 3 7 6 | 2 1 4 
4 1 2 | 8 9 5 | 7 3 6 
7 6 3 | 4 1 2 | 9 8 5 
------+-------+------
6 9 1 | 2 8 4 | 5 7 3 
8 2 5 | 1 3 7 | 4 6 9 
3 4 7 | 6 5 9 | 8 2 1 

Algorithm: BFS
Total time: 380 ms
Total expanded states: 58415
